pyo3-ffi/src/cpython/code.rs

```diff
@@ -232,6 +232,7 @@ pub const CO_MAXBLOCKS: usize = 20;
 
 #[cfg_attr(windows, link(name = "pythonXY"))]
 extern "C" {
+    #[cfg_attr(PyPy, link_name = "PyPyCode_Type")]
```

Comment:
I couldn't find `PyPyCode_Type` in https://github.com/PyO3/python3-dll-a/blob/main/src/libpypy3.10-c.def

Reply:
Thanks! So, I guess we should disable `PyCode` on PyPy